articles/extensions/logstash.md

```diff
@@ -4,33 +4,43 @@ description: How to install and configure the Auth0 Logs to Logstash extension.
 
 # Auth0 Logs to Logstash
 
-The _Auth0 Logs to Logstash_ is a scheduled job that takes all of your Auth0 logs and exports them to [Logstash](https://www.elastic.co/products/logstash). Logstash is an open source log management tool that is most often used as part of the ELK stack along with ElasticSearch and Kibana.
+The **Auth0 Logs to Logstash** is a scheduled job that takes all of your Auth0 logs and exports them to [Logstash](https://www.elastic.co/products/logstash). Logstash is an open source log management tool that is most often used as part of the ELK stack along with ElasticSearch and Kibana.
 
-## Configuring the Extension
+## Configure the Extension
 
 To install and configure this extension, click on the _Auth0 Logs to Logstash_ box in the list of provided extensions on the [Extensions](${manage_url}/#/extensions) page of the [Management Portal](${manage_url}). The _Install Extension_ window pops open.
 
-![](/media/articles/extensions/logstash/extension-mgmt-logstash.png)
+![Dashboard > Logstash Extension](/media/articles/extensions/logstash/extension-mgmt-logstash.png)
 
 At this point you should set the following configuration variables:
 
-- **Schedule**: The frequency with which logs should be exported. The schedule can be customized even further after creation.
-- **BATCH_SIZE**: The ammount of logs to be read on each execution. Maximun is 100.
-- **LOGSTASH_URL**: Your Logstash URL as defined for use with `logstash-input-http` plugin.
-- **LOGSTASH_INDEX**: Your Logstash Index to which the logs will be routed.
-- **LOG_LEVEL**: The minimal log level of events that you would like sent to Logstash.
-- **LOG_TYPES**: The events for which logs should be exported.
+| Parameter        | Description |
+|:-----------------|:------------|
+| **Schedule** | The frequency with which logs should be exported. The schedule can be customized even further after creation. |
+| **BATCH_SIZE** | The ammount of logs to be read on each execution. Maximun, and default, is `100`. |
+| **LOGSTASH_URL** <br/><span class="label label-danger">Required</span> | Your Logstash URL as defined for use with `logstash-input-http` plugin. |
+| **LOGSTASH_INDEX** <br/><span class="label label-danger">Required</span> | Your Logstash Index to which the logs will be routed. |
+| **LOGSTASH_TOKEN** | The token required for your Logstash deployments that will be included in the querystring. |
+| **LOGSTASH_USER** | The Logstash user. |
+| **LOGSTASH_PASSWORD** | The password associated with your Logstash user. |
+| **START_FROM** | The checkpoint ID of the log from where you want to start. |
```

Comment:
@khorne3 Hey I changed the format from bullets to table, I think it looks better that way, what do you think?

Also, about `START_FROM` I cannot see it in my dashboard, is this released?